src/main/java/edu/cornell/kfs/module/cam/CuCamsConstants.java

```diff
@@ -6,12 +6,13 @@
     }
 
     public static final class CapAssetApi {
+        public static final String COGNITO_USER_POOL_ISSUER_URL = "https://cognito-idp.us-east-1.amazonaws.com/us-east-1_PF3qpyCGu";
         public static final String OBJECT_NOT_FOUND = "Object not found";
         public static final String ERROR = "error";
-        public static final String CAPITAL_ASSET_CREDENTIAL_GROUP_CODE = "ASSETSCAN";
         public static final String UNAUTHORIZED = "Unauthorized";
         public static final String BAD_REQUEST = "Invalid Request";
-        public static final String CAPITAL_ASSET_API_KEY_CREDENTIAL_NAME = "capital_asset_scanning_api_key";
+        public static final String COGNITO_ID_TOKEN = "cognito_id_token";
+        public static final String COGNITO_PUBLIC_KEY_FILE_RELATIVE_PATH = "../webapps/kfs/WEB-INF/classes/edu/cornell/kfs/module/cam/capital_asset_inventory_cognito_public_key.json";
```

Comment:
I have moved this to the kfs security configuration file (cu.kfs.cams.api.public.key.json), also I have listed the public key json directly rather than the file path